### PR TITLE
UG for Custom-Bookmark feature

### PR DESCRIPTION
**Pull request by SalmanBaris25**
UG for Custom-Bookmark feature

### CODE REVIEW

File: MAUI/PDF-Viewer/Custom-Bookmark.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+title: Custom Bookmark in .NET MAUI PDF Viewer control | Syncfusion
+description: Learn here about the custom bookmark navigation in Syncfusion .NET MAUI PDF Viewer (SfPdfViewer) control.
+platform: MAUI
+control: SfPdfViewer
+documentation: ug
+---
+
+# Custom bookmark in .NET MAUI PDF Viewer
+
+A PDF document may optionally have custom bookmarks that allow the user to bookmark pages and navigate them. The PDF viewer control displays these custom bookmarks in a list of page elements.
```
Comment (NavaneethS) [inline MAUI/PDF-Viewer/Custom-Bookmark.md:12]: navigate to them. 
"list of page elements"?

File: MAUI/PDF-Viewer/Custom-Bookmark.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+{% highlight c# %}
+
+pdfViewer.IsOutlineViewVisible = true;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Accessing custom bookmarks collection
+
+To access the custom bookmarks, you can use the `CustomBookmarks` collection. This property provides a list of custom booknarks.
```
Comment (NavaneethS) [inline MAUI/PDF-Viewer/Custom-Bookmark.md:33]: the list of custom bookmarks in the PDF.

File: MAUI/PDF-Viewer/Custom-Bookmark.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+{% endtabs %}
+
+## Editing a Custom Bookmark
+
+To edit an existing custom bookmark, you can retrieve the bookmark from the `CustomBookmarks` collection and modify its properties:
+
+{% tabs %}
+{% highlight c# %}
+
+Bookmark bookmarkToEdit = pdfViewer.CustomBookmarks.FirstOrDefault(b => b.Title == "Bookmark to Edit");
```
Comment (NavaneethS) [inline MAUI/PDF-Viewer/Custom-Bookmark.md:73]: GIve a more meaning name such as "Chapter 2"

File: MAUI/PDF-Viewer/Custom-Bookmark.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+if (bookmarkToEdit != null)
+{
+    bookmarkToEdit.Name = "Edited Bookmark Title";
+    bookmarkToEdit.PageNumber = 1; // Change the page index
```
Comment (NavaneethS) [inline MAUI/PDF-Viewer/Custom-Bookmark.md:77]: PageNumber cannot be set.

File: MAUI/PDF-Viewer/Custom-Bookmark.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+
+
+{% endhighlight %}
+{% endtabs %}
+
+## Navigating to custom bookmark
+
+### Navigating using UI
+
+As mentioned above, you can show the outline view by setting the `IsOultineViewVisible` property to `true`. When the outline view is showing, you can tap on any bookmark to navigate to the destination pointed by that bookmark.
```
Comment (NavaneethS) [inline MAUI/PDF-Viewer/Custom-Bookmark.md:105]: "pointed to"

File: MAUI/PDF-Viewer/Custom-Bookmark.md [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+![Custom bookmark in .NET MAUI PDF Viewer](Images\custom-bookmark.png)
+
+### Navigating programmatically
+
+The PDF viewer allows the users to navigate to an custom bookmark using the `GoToBookmark` method. The below code snippet illustrates the same.
+
+{% tabs %}
+{% highlight c# %}
+
+//Get the required custom bookmark
```
Comment (M-A32) [inline MAUI/PDF-Viewer/Custom-Bookmark.md:131]: //Get the required custom bookmark.

File: MAUI/PDF-Viewer/Custom-Bookmark.md [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+
+### Navigating using UI
+
+As mentioned above, you can show the outline view by setting the `IsOultineViewVisible` property to `true`. When the outline view is showing, you can tap on any bookmark to navigate to the destination pointed to that bookmark. 
+
+![Custom bookmark in .NET MAUI PDF Viewer](Images\custom-bookmark.png)
+
+### Navigating programmatically
+
+The PDF viewer allows the users to navigate to an custom bookmark using the `GoToBookmark` method. The below code snippet illustrates the same.
```
Comment (M-A32) [inline MAUI/PDF-Viewer/Custom-Bookmark.md:126]: The PDF viewer allows the users to navigate to an custom bookmark using the `GoToBookmark` method. The following code sample illustrates the same.

**Review by NavaneethS — APPROVED** [review]

File: MAUI/PDF-Viewer/Custom-Bookmark.md [hunk 8]
```diff
@@ -0,0 +1,138 @@
...
+
+pdfViewer.CustomBookmarks.Add(newBookmark);
+
+
+{% endhighlight %}
+{% endtabs %}
+
+### Rename custom bookmarks using the bookmark context menu
+
+Tap the context menu button on the custom bookmark to be edited and choose Rename and enter the desired name.
```
Comment (M-A32) [inline MAUI/PDF-Viewer/Custom-Bookmark.md:74]: Tap the context menu button on the custom bookmark to be edited and then choose Rename and enter the desired name.

File: MAUI/PDF-Viewer/Custom-Bookmark.md [hunk 9]
```diff
@@ -0,0 +1,138 @@
+---
+layout: post
+title: Custom Bookmark in .NET MAUI PDF Viewer control | Syncfusion
```
Comment (M-A32) [inline MAUI/PDF-Viewer/Custom-Bookmark.md:3]: title: Custom Bookmark in .NET MAUI PDF Viewer Control | Syncfusion

**Review by M-A32 — CHANGES_REQUESTED** [review]
I have completed the review.

**Pull request merged**